code/game/turfs/turf.dm

```diff
@@ -108,7 +108,7 @@
 //		O.update_gravity(O.mob_has_gravity(src))
 
 	var/loopsanity = 100
-	for(var/atom/A in range(1))
+	for(var/atom/movable/A in range(1))
```

Comment:
... um, what

It's an /atom/proc
which takes an argument of an /atom/movable
this is correct, the for loop is *calling* HasProximity on every atom nearby, with the argument of what is proximate to them